packages/flutter/lib/src/material/button_style.dart

```diff
@@ -208,6 +209,16 @@ class ButtonStyle with Diagnosticable {
   ///  * [Feedback] for providing platform-specific feedback to certain actions.
   final bool? enableFeedback;
 
+  /// The alignment of the button's [child].
```

Comment:
This [child] is an invalid reference.  Either just `child` or I think you'll have to do [TextButton.child] or something like that.